File: marta_kotsupyr/bridge/Circle.cs
using System;

public class Circle : Shape
{
    private double radius;

    public Circle(IColor color, double radius) : base(color)
    {
        this.radius = radius;
    }

    public override double CalculateArea()
    {
        return Math.PI * radius * radius;
    }

    public override double CalculatePerimeter()
    {
        return 2 * Math.PI * radius;
    }
}

File: rad_anastasiya/bridge/src/com/universiry/bridge/shape/Circle.java
package com.universiry.bridge.shape;

import com.universiry.bridge.color.IColor;

public class Circle extends Shape {
    private final double radius;

    public Circle(final double radius, final IColor IColor) {
        super(IColor);
        this.radius = radius;
    }

    @Override
    public double square() {
        return Math.PI * Math.pow(radius, 2);
    }

    @Override
    public double perimeter() {
        return 2 * Math.PI * radius;
    }

    @Override
    public String toString() {
        return "Circle{" + "radius: " + radius
                + ", color: " + IColor.name()
                + ", perimeter: " + perimeter()
                + ", square: " + square()
                + '}';
    }
}
